dependency-download/build.gradle

```diff
@@ -20,8 +20,12 @@ repositories {
 }
 
 dependencies {
+<<<<<<< HEAD
     runtime group: 'dev.galasa', name: 'dev.galasa.managers.manifest', version: version
     runtime group: 'dev.galasa', name: 'dev.galasa.framework.manifest', version: version
+=======
+    runtime group: 'dev.galasa', name: 'dev.galasa.managers', version: version
+>>>>>>> main
 }
```

Comment:
is this correct? looks like merge conflict text?

Reply:
good catch. Resolved.